pkg/cmd/factory/http.go

```diff
@@ -127,6 +128,22 @@ func NewHTTPClient(io *iostreams.IOStreams, cfg configGetter, appVersion string,
 	return api.NewHTTPClient(opts...), nil
 }
 
+var ssoHeader string
+var ssoURLRE = regexp.MustCompile(`\burl=(\S+)`)
```

Comment:
Considering the possibility of adding values after the url, wouldn't this regular expression be better? 😃 
```
\burl=([^;]+)
```
https://rubular.com/r/oAROq4p1c20WCu

Reply:
That's a good thought, but I wonder whether a literal `;` would ever be present in the URL? Since GitHub uses `&` for param delimiter, I would guess that it's unlikely that we ever see a literal `;`, in which case your proposed regex would be more future-proof.

Reply:
Sorry, I was not concerned about the url param, but about the value being added to X-GitHub-SSO.
But if there are no such changes in the future, it seems to be fine. 😃